modules/mobile/spec/requests/mobile/v0/appointments/create_spec.rb

```diff
@@ -16,6 +16,7 @@
   let!(:user) { sis_user(icn: '1012846043V576341') }
 
   before do
+    allow_any_instance_of(User).to receive(:va_patient?).and_return(true)
```

Comment:
just adding the facilities was not working in this spec as it was transforming facilities into test facilities 984. This change is unique to testing env so not worried about effecting prod performance.